espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/ReleaseActions.kt

```diff
@@ -0,0 +1,23 @@
+package io.appium.espressoserver.lib.handlers
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import io.appium.espressoserver.lib.helpers.w3c.models.Actions
+import io.appium.espressoserver.lib.model.AppiumParams
+import io.appium.espressoserver.lib.viewaction.UiControllerPerformer
+import io.appium.espressoserver.lib.viewaction.UiControllerRunnable
+
+
+class ReleaseActions : RequestHandler<AppiumParams, Void?> {
+
+    @Throws(AppiumException::class)
+    override fun handle(params: AppiumParams): Void? {
+
+        val runnable = UiControllerRunnable<Void> {
```

Comment:
might be way cleaner with lambdas

Reply:
I'm going to leave a `TODO` for this.... I'll need to understand the lambda -> runnable relation better first. In the future, I can do a PR to convert _all_ runnables to lambda.